db/migrate/20180810162637_delete_assignments_with_duplicate_mission_and_user.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class DeleteAssignmentsWithDuplicateMissionAndUser < ActiveRecord::Migration[5.1]
+  def up
+    duplicates = []
+    dup_attrs = Assignment.select(:mission_id, :user_id).group(:mission_id, :user_id).having("count(*) > 1")
+
+    dup_attrs.each do |dup|
+      dup_assignments = Assignment.where(mission_id: dup.mission_id, user_id: dup.user_id)
+      duplicates << dup_assignments[1..-1]
+      duplicates.flatten.each(&:destroy)
```

Comment:
The way this is written it will destroy the first element over and over again.

You can simply do `Assignment.where(mission_id: dup.mission_id, user_id: dup.user_id)[1..-1].each(&:destroy)` or some such.